### ISSUE TITLE
Couldn't find this error anywhere

### ISSUE THREAD

**Issue by DumbMahreeo** [chromedp/chromedp#765]
#### What versions are you running?

<pre>
github.com/chromedp/chromedp v0.6.6
Chromium 89.0.4389.72 snap
go version go1.16 linux/amd64
</pre>

#### What did you do? Include clear steps.
```go
    ctx, cancel := chromedp.NewContext(context.Background())
    defer cancel()

    timectx, cancel := context.WithTimeout(ctx, timeout)
    defer cancel()

    var price string
    var avail string

    fmt.Println(colorscheme["blue"] + "Amazon")
    err := chromedp.Run(
        timectx,
        chromedp.Navigate(link),
        chromedp.Text(`[data-action="show-all-offers-display"]`, &avail, chromedp.ByQuery),
        chromedp.Text(`#priceblock_ourprice`, &price, chromedp.ByID),
    )

    if err != nil {
        return false
   }
```

#### What did you expect to see?
Not an error

#### What did you see instead?
ERROR: unhandled page event *page.EventDocumentOpened

#### Did I try searching online
Yes, I couldn't find what causes this error, I can't find it anywhere, maybe it's just me, but I can't figure out why this happens. It's not something that happens always, it just randomly happens

**Comment by kenshaw** [comment]
Try updating to the latest `cdproto` definitions:

```sh
# go get -u github.com/chromedp/cdproto
```

**Comment by kenshaw** [comment]
My mistake. This is an experimental event that was added -- see: https://chromedevtools.github.io/devtools-protocol/tot/Page/#event-documentOpened

Note: you can safely ignore this, as it has no effect. I'll be pushing a change shortly that ignores this event in the page handler.

**Comment by kenshaw** [comment]
This has been fixed with `v0.6.8`.
